File: Aula03/app/src/main/java/br/com/nglauber/aula03/Pessoa.java
package br.com.nglauber.aula03;

import org.parceler.Parcel;

@Parcel
public class Pessoa {

    public static final int RS_FACEBOOK = 0;
    public static final int RS_TWITTER = 1;
    public static final int RS_GPLUS = 2;

    public String nome;
    public int redesocial;

    @Override
    public String toString() {
        return nome +" - "+ redesocial;
    }
}

File: Aula12/app/src/main/java/br/com/nglauber/aula12/Pessoa.kt
package br.com.nglauber.aula12

import org.parceler.Parcel

@Parcel
data class Pessoa (var uid: String = "",
                   var nome: String = "",
                   var redesocial: Int = RS_FACEBOOK) {

    companion object {
        val RS_FACEBOOK = 0
        val RS_TWITTER = 1
        val RS_GPLUS = 2
    }
}
